protocol/messenger_news_feed_test.go

```diff
@@ -0,0 +1,56 @@
+package protocol
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/mmcdole/gofeed"
+
+	multiaccountscommon "github.com/status-im/status-go/multiaccounts/common"
+)
+
+type MessengerNewsFeedSuite struct {
+	CommunitiesMessengerTestSuiteBase
+	m *Messenger // main instance of Messenger
+}
+
+func (s *MessengerNewsFeedSuite) SetupTest() {
+	s.CommunitiesMessengerTestSuiteBase.SetupTest()
+
+	s.m = s.newMessenger(alicePassword, []string{aliceAccountAddress})
+	s.m.account.CustomizationColor = multiaccountscommon.CustomizationColorOrange
+
+	_, err := s.m.Start()
+	s.Require().NoError(err)
+}
+
+func TestMessengerNewsFeedSuite(t *testing.T) {
+	suite.Run(t, new(MessengerNewsFeedSuite))
+}
+
+func ptrTime(t time.Time) *time.Time {
+	return &t
+}
+
+func (s *MessengerNewsFeedSuite) TestHandleNewsFeedItem() {
+	item := gofeed.Item{
+		GUID:            "12345",
+		Title:           "New Item",
+		PublishedParsed: ptrTime(time.Now().Add(-1 * time.Hour)),
+		Description:     "This is a new item",
+		Link:            "https://example.com/new-item",
+		Content:         "This is the content of the new item",
+		Image:           &gofeed.Image{URL: "https://example.com/image.jpg"},
```

Comment:
It's better to use some fake generated data. 
This makes the intentions explicit and prevents any potential collision with hard-coded data.
(not a big deal, just a suggestion for the future)
```suggestion
		GUID:            gofakeit.UUID()
		Title:           gofakeit.LetterN(5),
		PublishedParsed: ptrTime(time.Now().Add(-1 * time.Hour)),
		Description:     gofakeit.Letter(5),
		Link:            gofakeit.URL(),
		Content:         gofakeit.LetterN(5),
		Image:           &gofeed.Image{URL: gofakeit.URL()},
```